core/include/subsystems/lift.h

```diff
@@ -22,13 +22,13 @@ class Lift
 {
   public:
 
-  struct lift_cfg_t
+  typedef struct
   {
     double up_speed, down_speed;
     double softstop_up, softstop_down;
 
     PID::pid_config_t lift_pid_cfg;
-  };
+  }lift_config_t;

```

Comment:
Do we think we should avoid changing names of structs for the sake of "backwards compatibility" (so that 2021-2022-BIG and stuff like that can still build)? Or does it not matter since we don't have those robots anymore? I could see it going either way

Reply:
I think because of the way git subrepo works, the version that compiles is still in the 2021-2022-BIG repo so I don't see a huge issue with this change. Only situation I can see it cause an issue is if an old Nemo repo (brick city) needed to be updated, but even then it's a small change